influxdb2/umbrel-app.yml

```diff
@@ -0,0 +1,37 @@
+manifestVersion: 1
+id: influxdb2
+name: InfluxDB2
+tagline: The leading platform for time series data
+icon: ""
```

Comment:
This key-value pair can be entirely deleted

```suggestion
```

Reply:
I've changed this here https://github.com/getumbrel/umbrel-apps/pull/2008/commits/a877dd6a6f9b71870c9eb419b36a365038f8f21c